docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,108 @@
+# Add conditions to TODO comments to make them error
```

Comment:
Whether it *errors* or *warns* is up to the user, so the title needs some improvement.

Reply:
Done